utils/python/codal_utils.py

```diff
@@ -15,24 +15,11 @@ def system(cmd):
       sys.exit(1)
 
 def build(clean, verbose = False):
-    if platform.system() == "Windows":
-        # configure
-        system("cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo -G \"Ninja\"")
+    # configure
+    system("cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo -DMANUAL_COMPILER_INCLUDE=true -G \"Ninja\"")
 
-        # build
-        system("ninja")
-    else:
-        # configure
-        system("cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo -G \"Unix Makefiles\"")
-
-        if clean:
-            system("make clean")
-
-        # build
-        if verbose:
-            system("make -j 10 VERBOSE=1")
-        else:
-            system("make -j 10")
```

Comment:
This will add the ninja dependency to the macOS and linux environment, which might be unexpected, so maybe we want to keep that (worth checking with Joe/James).

Apart from that it looks like we are looking the verbose option?

Reply:
I'll ask Joe about that, then. 

> Apart from that it looks like we are looking the verbose option?
You're right - easy to add back as ninja has essentially the same launch commands as make.